docs/suspensive.org/src/pages/docs/react-query/_meta.en.json

```
@@ -1,7 +1,7 @@
 {
   "motivation": { "title": "Why need to use?" },
   "installation": { "title": "Installation" },
-  "command-line-interface": { "title": "Command-Line Interface" },
+  "support-both-tanstack-query-v4-and-5": { "title": "Support both TanStack Query v4 and 5" },
```

There may be a time in the future to support v6, so I hope it is an expandable path name.
I think it's a good idea for links not to change frequently

For example, when adding this link to a user's blog, I would like to prevent the user from being unable to enter if the link changes.

```suggestion
  "tanstack-query-compatibility": { "title": "Support both TanStack Query v4 and 5" },
```

---

Oh, I hadn't thought of that. Thank you!